src/main/java/com/iciql/TableInspector.java

```diff
@@ -115,6 +115,7 @@ void read(DatabaseMetaData metaData) throws SQLException {
             while (rs.next()) {
                 IndexInspector info = new IndexInspector(rs);
                 if (info.type.equals(IndexType.UNIQUE)) {
+                    if(info.name == null) continue;
```

Comment:
The concept of the fix is good but I'm not crazy about the proposed one-liner; that's not my preferred style.

Consider this alternative:
```java
if (info.name != null && info.type.equals(IndexType.UNIQUE)) {
```